backend/tests/modules/v2/test_allocations_v2.py

```diff
@@ -0,0 +1,462 @@
+from datetime import datetime
+import pytest
+from httpx import AsyncClient
+from sqlalchemy.ext.asyncio import AsyncSession
+
+from tests.modules.v2.utils import FakeAllocationRequest, FakeUser, FakeAllocation
+
+
+class TestGetDonorsForEpochV1:
+    """Test cases for the GET /allocations/donors/{epoch_id} (get_donors_for_epoch_v1) endpoint"""
+
+    @pytest.mark.asyncio
+    async def test_returns_empty_list_when_no_donors(
+        self, fast_client: AsyncClient, fast_session: AsyncSession
+    ):
+        """Should return an empty list when there are no donors for the epoch"""
+
+        async with fast_client as client:
+            # When: requesting donors for epoch 1
+            resp = await client.get("allocations/donors/1")
+
+            # Then: response should be successful with empty donors list
+            assert resp.status_code == 200
```

Comment:
maybe le'ts unify it to using `status.HTTP_200_OK` etc. all such hardcoded status_code

Reply:
yes